client/app/scripts/components/grid-mode-selector.js

```diff
@@ -4,6 +4,21 @@ import classNames from 'classnames';
 
 import { toggleGridMode } from '../actions/app-actions';
 
+
+function renderItem(icons, label, isSelected, onClick) {
```

Comment:
I think we should distinguish functions that return JSX a bit more, to make it visible that they return UI components. I have seen this format in other projects:
```
const Item = (icons, label, isSelected, onClick) => (
  <div>...</div>
);
```
Notice the first letter capitalized, like it is for component classes. That makes it easy to read when used in other places, e.g., `{items.map(Item)}`